v21.2/performance-best-practices-overview.md

```diff
@@ -323,53 +323,69 @@ If you have long-running queries (such as analytics queries that perform full ta
 
 However, because `AS OF SYSTEM TIME` returns historical data, your reads might be stale.
 
-## Understanding and avoiding transaction contention
+## Hot spots
 
-Transaction contention occurs when the following three conditions are met:
+Transactions that operate on the same range but _different index key values_ will be limited by the overall hardware capacity of a single node (the range lease holder). These are referred to as _hot spots_.
```

Comment:
As you did with 'Indicators of Transaction Contention' below, I think it would be good to add a similar subsection here and point users to the [hot ranges API endpoint](https://www.cockroachlabs.com/docs/v21.2/cluster-api#resources) (see `/ranges/hot`)

Reply:
Fixed.